platform/ios/test/MGLMapViewPitchTests.m

```diff
@@ -0,0 +1,156 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+static MGLMapView *mapView;
```

Comment:
Any particular reason why these are globals rather than test properties?

Reply:
There’s no reason for this to be `static` globally — I copy/pasted this from another test. I do like not having to type out `self.mapView`, but that’s not a meaningful improvement, so I made this a property on `MGLMapViewPitchTests` in https://github.com/mapbox/mapbox-gl-native/pull/15028/commits/322dce36db797098e4ae5e9b540a9a54bbddc1e1.